tests/integration/config/plugins/integration-tests/data/tests/tests_flagship_planet_attribute.txt

```diff
@@ -0,0 +1,122 @@
+test-data "flagship planet attribute missions"
+	category "savegame"
+	contents
+		pilot Bobbi Bughunter
+		date 16 11 3013
+		system "Terra Incognita"
+		planet Ruin
+		clearance
+		ship "Star Barge"
+			name "Buggy Barge"
+			sprite "ship/star barge"
+			attributes
+				category "Light Freighter"
+				cost 190000
+				mass 10
+				bunks 3
+				"cargo space" 50
+				drag 1
+				"engine capacity" 400
+				"fuel capacity" 300
+				"heat dissipation" 0.8
+				hull 1000
+				"outfit space" 1300
+				"required crew" 1
+				shields 600
+				"turret mounts" 1
+				"weapon capacity" 200
+				"thrust" 50
+				"turn" 1000
+			outfits
+				Hyperdrive
+				"nGVF-BB Fuel Cell"
+				"LP036a Battery Pack"
+			crew 1
+			fuel 300
+			shields 600
+			hull 1000
+			position 0 0
+			engine -9 38 1
+			engine 9 38 1
+			turret 0 -8
+			leak leak 60 50
+			explode "tiny explosion" 10
+			explode "small explosion" 10
+			system "Terra Incognita"
+			planet Ruin
+		account
+			credits 100000000
+			score 400
+			history
+		visited "Terra Incognita"
+		"visited planet" Ruin
+		conditions
+			# to skip the landing on ruin conversation
+			"Ruin: Landing: offered"
+		changes
+			planet Ruin
+				add attributes "Temperate Climate"
+
+
+
+mission "%TEST%: FLAGSHIP PLANET ATTRIBUTE MISSION"
+	source "Ruin"
+	to offer
+		has "%TEST%: FLAGSHIP PLANET ATTRIBUTE MISSION OFFER"
+		has "flagship planet attribute: Temperate Climate"
+	on offer
+		conversation
+			`THIS IS A FLAGSHIP PLANET ATTRIBUTE MISSION YO!`
+				decline
+	on decline
+		set "succeed test"
+
+
+
+test "Trigger Flagship Planet Attribute Mission"
+	status partial
+	description "Launch, land, and go to the spaceport to trigger any potential missions in the shipyard."
+	sequence
+		call "Depart"
+		call "Land On Ruin helper"
+		input
+			key p
+
+
+
+test "End Flagship Planet Attribute Missions"
+	status partial
+	description "Just skip a mission by pressing return twice."
+	sequence
+		# Press accept (or decline, depending on if it is shown) and then skip the dialog once.
+		input
+			key "Return"
+		input
+			key "Return"
+
+
+test "Test Flagship Planet Attribute Mission"
+	status partial
+	description "Set a few conditions so they are displayed on errors, clarifying things up."
+	sequence
+		apply
+			# Test accepted. If not there it wasn't offered.
+			"test: succeed test" = "succeed test"
+			# if we launched
+			"test: flagship planet: Ruin" = "flagship planet: Ruin"
+
+
+test "Flagship Planet Attribute Mission Test"
+	status active
+	description "Test if a conversation option shows up when the given condition is set."
+	sequence
+		inject "flagship planet attribute missions"
+		call "Load First Savegame"
+		apply
+			"%TEST%: FLAGSHIP PLANET ATTRIBUTE MISSION OFFER" = 1
+		call "Trigger Flagship Planet Attribute Mission"
+		call "End Flagship Planet Attribute Missions"
+		call "Depart"
+		call "Test Shipyard And Outfitter Mission"
+		assert
+			"succeed test" == 1
```

Comment:
this is a bit weird when we could just test for `"flagship planet attribute: Temperate Climate"` directly

Reply:
I still want to test via a mission so it’s a full integration test but can definitely add these other asserts.

Reply:
Add a true assert and a false assert. Kept the mission structure for a full end to end test, but can remove that part if ya'll would prefer brevity here.

Reply:
Idk, testing via a mission doesnt seem very useful and requires one more landing depart, making the test take twice as long so I'd be for brevity.
It's not use to test if the condition is accessible by missions in this test, if we do a test for that it should be bigger and take all of them into account, this one should just focus on flagship planet attribute being set

Reply:
Trimmed it down, I think that works